x-pack/plugins/security_solution/public/common/containers/matrix_histogram/index.ts

```diff
@@ -240,3 +244,72 @@ export const useMatrixHistogram = ({
 
   return [loading, matrixHistogramResponse, runMatrixHistogramSearch];
 };
+
+/* function needed to split ip histogram data requests due to elasticsearch bug https://github.com/elastic/kibana/issues/89205
+ * using includeMissingData parameter to do the "missing data" query separately
+ **/
+export const useMatrixHistogramCombined = (
```

Comment:
I created this hook to encapsulate the "hacky" code, when the elasticsearch bug is fixed we can just remove this hook and go back and use the plain useMatrixHistogram in the component.